packages/riipen-ui/src/components/editor/BlockStyleControls.jsx

```diff
@@ -0,0 +1,105 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faListOl,
+  faListUl,
+  faQuoteLeft
+} from "@fortawesome/free-solid-svg-icons";
```

Comment:
We originally used the pro icons in the original RTE, not sure how to handle that.

Reply:
I think this is actually better. Helps consolidate all of our iconography.